elyra/kfp/tests/test_bootstrapper.py

```diff
@@ -54,7 +54,11 @@ def start_minio():
             check=True)
         time.sleep(3)
     except subprocess.CalledProcessError as ex:
-        raise RuntimeError(f'Error executing process: {ex.stderr.decode("unicode_escape")}') from ex
+        if ex.stderr:
+            error = ex.stderr.decode("unicode_escape")
```

Comment:
I'm fairly certain we can abandon the `.decode("unicode_escape")` suffix altogether since all strings are unicode in Python 3 anyway.  `decode()` is essentially unnecessary.

As a result, we could revert to the original statement with something like:
```python
raise RuntimeError(f"Error executing process: {ex.stderr or '')}") from ex
```

A quick test of this yields the following in the output...
```
        except subprocess.CalledProcessError as ex:
>           raise RuntimeError(f"Error executing process: {ex.stderr or ''}") from ex
E           RuntimeError: Error executing process:
```
which indicates that `ex.stderr` was `None`.

Reply:
If that's ok then I'll update it to that then

Reply:
pushed